image-processing.py

```diff
@@ -51,6 +51,43 @@ def mirrorLeftToRight(width, height, step, que):
 				print("INDEX ERROR AT:", (x,y))
 				exit()	
 
+def lapalacian(width, height, step, que):
+	#discrete laplace
+	global newImg
+	global currentPix
+	global newPix
+	
+	kernel = ((0,1,0),(1,-4,1),(0,1,0))
+	for x in range(width):
+		for y in range(height):
+			r = 0
+			g = 0
+			b = 0
+			for xb in range(0,3):
+				for yb in range(0,3):
+					if kernel[xb][yb] != 0:
+						pixel = getPixelWithOverlap(x+xb-1, y+yb-1, width, height)
```

Comment:
Your documentation has been very useful for following each step of code you have written. Can you explain what the getPixelWithOverlap function is doing on line 69?